ansible/playbook.yml

```diff
@@ -136,6 +136,13 @@
       tags:
         - aws-only
 
+    # Install salt minion
+    # Only for AWS images
+    - name: install salt minion
```

Comment:
better spot to import these might be in ansible/tasks/setup-supabase-internal.yml

Reply:
Moved tasks